### PR TITLE
Implement storybook#4

### PR DESCRIPTION
**Pull request by amagin006**

### CODE REVIEW

File: stories/0-Welcome.stories.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+export default {
+  title: 'Welcome',
+};
+
+export const toStorybook = () => <Welcome showApp={linkTo('Button')} />;
+
+toStorybook.story = {
+  name: 'to Storybook',
+};
```
Comment (Saayaman) [inline stories/0-Welcome.stories.js]: we can delete this example because it is confusing

File: stories/1-Button.stories.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+export const text = () => <Button onClick={action('clicked')}>Hello Button</Button>;
+
+export const emoji = () => (
+  <Button onClick={action('clicked')}>
+    <span role="img" aria-label="so cool">
+      😀 😎 👍 💯
+    </span>
+  </Button>
+);
```
Comment (Saayaman) [inline stories/1-Button.stories.js]: we can also delete this one and just keep CardExample as eample

File: stories/0-Welcome.stories.js (hunk 1 above)
Comment (amagin006) [reply stories/0-Welcome.stories.js]: Deleted this file

File: stories/1-Button.stories.js (hunk 2 above)
Comment (amagin006) [reply stories/1-Button.stories.js]: Deleted this file

**Pull request merged**